### PR TITLE
Update TestNG to 7.5.1

### LINKED ISSUE #23781: Update to TestNG 7.5.x
https://repo1.maven.org/maven2/org/testng/testng/7.7.1/

### PR DESCRIPTION
**Pull request by abutch3r**
Updated Arquillian related variables to improve consistency and findability across individual projects when updates are to be made.

Update arquillian versions to latest known tested versions that should be compatible with TestNG 7.5.x

7.5.x of TestNG now requires SLF4J, so update ArquillianLoadableExtension to include SLF4J.

fixes #23781, #25109

### CODE REVIEW

**Comment by abutch3r** [comment]
#libby
#build

File: dev/cnf/oss_ibm.maven [hunk 1]
```diff
@@ -70,4 +70,5 @@ org.eclipse.microprofile.rest.client:microprofile-rest-client-tck:1.4.2-20210215
 org.mock-server:mockserver-core:3.10.7-IBM20191022
 org.mock-server:mockserver-core:3.10.7-IBM20191022
 org.openid4java:openid4java:0.9.7-ibm-s20130624-1827
+org.testng:testng:7.5.1
```
Comment (tevans78) [inline dev/cnf/oss_ibm.maven]: Need to move this back to the `oss_dependencies.maven` file now that it is in Maven Central rather than DHE

**Comment by abutch3r** [comment]
#libby
#build

**Comment by abutch3r** [comment]
dhe check was due to incorrect location of the testng in .dependencies files and has been resolved by restoring to original location

Test failures were due to infrastructure(docker) and a known unrelated test case failure

**Review by tevans78 — DISMISSED** [review]

**Review by tevans78 — APPROVED** [review]

**Comment by abutch3r** [comment]
#build

**Comment by abutch3r** [comment]
#build

**Comment by abutch3r** [comment]
#libby

**Comment by abutch3r** [comment]
The failing TCK bucket is a intermittent failure on the stopping of the server, all the tests still passed in the bucket

Other failing bucket has since been quarantined in the main build

**Pull request merged**